share/tasks.py

```diff
@@ -1,409 +1,188 @@
-
-            if force and error:
-                log.forced(error)
-            else:
-                log.succeed()
-
-
-class NormalizerTask(SourceTask):
-
-    def do_run(self, raw_id):
-        raw = RawDatum.objects.get(pk=raw_id)
-        transformer = self.config.get_transformer()
-
-        assert raw.suid.source_config_id == self.config.id, '{!r} is from {!r}. Tried parsing it as {!r}'.format(raw, raw.suid.source_config_id, self.config.id)
-
-        logger.info('Starting normalization for %s by %s', raw, transformer)
-
-        try:
-            graph = transformer.transform(raw)
-
-            if not graph or not graph['@graph']:
-                logger.warning('Graph was empty for %s, skipping...', raw)
-                return
-
-            normalized_data_url = settings.SHARE_API_URL[0:-1] + reverse('api:normalizeddata-list')
-            resp = requests.post(normalized_data_url, json={
-                'data': {
-                    'type': 'NormalizedData',
-                    'attributes': {
-                        'data': graph,
-                        'raw': {'type': 'RawData', 'id': raw_id},
-                        'tasks': [self.task.id]
-                    }
+        normalized_data_url = settings.SHARE_API_URL[0:-1] + reverse('api:normalizeddata-list')
+        resp = requests.post(normalized_data_url, json={
+            'data': {
+                'type': 'NormalizedData',
+                'attributes': {
+                    'data': graph,
+                    'raw': {'type': 'RawData', 'id': raw_id},
+                    'tasks': [CeleryTaskResult.objects.get(task_id=self.request.id).id]
                 }
-            }, headers={'Authorization': self.source.authorization(), 'Content-Type': 'application/vnd.api+json'})
-        except Exception as e:
-            logger.exception('Failed normalizer task (%s, %d)', self.config.label, raw_id)
-            raise self.retry(countdown=10, exc=e)
-
-        if (resp.status_code // 100) != 2:
-            raise self.retry(countdown=10, exc=Exception('Unable to submit change graph. Received {!r}, {}'.format(resp, resp.content)))
-
-        logger.info('Successfully submitted change for %s', raw)
-
-    def log_values(self):
-        return {
-            **super().log_values(),
-            'app_label': self.config.label,
-            'app_version': self.config.transformer.version,
-        }
+            }
+        }, headers={'Authorization': raw.suid.source_config.source.user.authorization(), 'Content-Type': 'application/vnd.api+json'})
+    except Exception as e:
+        logger.exception('Failed to transform %r', raw)
+        raise self.retry(countdown=10, exc=e)
+
+    if (resp.status_code // 100) != 2:
+        raise self.retry(countdown=10, exc=Exception('Unable to submit change graph. Received {!r}, {}'.format(resp, resp.content)))
+
+    logger.info('Successfully submitted change for %s', raw)
+
+
+@celery.shared_task(bind=True, retries=5)
```

Comment:
Should this be [`max_retries`](http://docs.celeryproject.org/en/latest/userguide/tasks.html#Task.max_retries)?

Also, `disambiguate` never calls `retry`; it looks like it'll give up after failing once.